Makefile

```diff
@@ -1,6 +1,8 @@
 .DEFAULT_GOAL = help
 
-NAME = storybook-dev
+REAL_ENV = dev
```

Comment:
Refs https://github.com/libero/storybook/pull/2#discussion_r328494178.

(https://github.com/libero/storybook/pull/4 will be updated to allow `ENV` to be set to `prod` on the host.)